apps/admin-ui/src/spa/ReservationUnit/edit/queries.ts

```diff
@@ -76,13 +73,6 @@ export const RESERVATIONUNIT_QUERY = gql`
         imageType
         imageUrl
       }
-      location {
-        addressStreetFi
-        addressZip
-        addressCityFi
-        longitude
-        latitude
-      }
```

Comment:
Check that this was removed logically (not needed anymore).

Reply:
Yeah. It's just done stupidly. There is a separate query on that page that fetches the unit and it's parameters with a pk. Could combine the queries into single one.